src/melee/gr/grtyoshi.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _grtyoshi_h_
+#define _grtyoshi_h_
+
+#include <global.h>
+#include <melee/gr/stage.h>
+#include <sysdolphin/baselib/gobj.h>
+
+static void func_80223B48(long);
+static void lbl_80223B4C(void);
+static void lbl_80223BBC(void);
+static void lbl_80223BC0(void);
+static s32 lbl_80223BE4(void);
+static HSD_GObj* func_80223BEC(s32 gobj_id);
+static void lbl_80223CD4(HSD_GObj* gobj);
+static s32 lbl_80223D00(void);
+static void lbl_80223D08(HSD_GObj*);
+static void lbl_80223D0C(void);
+static void lbl_80223D10(HSD_GObj* gobj);
+static s32 lbl_80223D60(void);
+static void lbl_80223D68(HSD_GObj* gobj);
+static void lbl_80223D9C(void);
+static void lbl_80223DA0(HSD_GObj* gobj);
+static s32 lbl_80223DF0(void);
+static void lbl_80223DF8(HSD_GObj*);
+static void lbl_80223E18(void);
+static int lbl_80223E1C(long);
```

Comment:
Please use the fixed-size primitive types in the future unless for a specific reason. This would normally be:
```suggestion
static BOOL lbl_80223E1C(s32);
```

It doesn't affect the match, so I'm not worried about it for this PR, just letting you know.